elytron/WFCORE-3947-ocsp.adoc

```diff
@@ -0,0 +1,67 @@
+= WFCORE-3947 OCSP support in Elytron SSL truststore
+:author:            Jan Kalina
+:email:             jkalina@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          elytron
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Elytron provides trustmanager usable for SSL contexts used by other parts of WildFly.
+It supports certificate revocations using certificate revocations list (CRL), but support of Online Certificate Status Protocol (OCSP) is required too.
+This is required especially for client certificates validation.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-715[EAP7-715]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFCORE-3947[WFCORE-3947]
+
+=== Dev Contacts
+
+* mailto:jkalina@redhat.com[Jan Kalina]
+
+=== QE Contacts
+
+* mailto:jstourac@redhat.com[Jan Stourac]
+
+=== Affected Projects or Components
+
+* WildFly Core
+* WildFly Elytron
+
+//=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* OCSP will be supported as an alternative to CRL in Elytron subsystem trustmanager (for client certificates verification).
+* Should be possible to specify OCSP responder manually, ignoring responder defined in certificates, for needs of creating caching responders/mirrors.
```

Comment:
Do I understand correctly that in case we don't have an OCSP responder setup manually, the caching would not be available? Or maybe I just don't completely understand this bullet.

Reply:
This is related to network-level caching - OCSP server in the datacenter working as proxy for multiple servers. When not set, URL of responder defined directly in the certificate is used.